scheduler/planner.go

```diff
@@ -223,6 +227,10 @@ func (unit *Unit) RankValue() int64 {
 		// would also be scheduled in a version.
 		priority += length
 	}
+	if generateTask {
+		// give generators a boost so people don't have to wait twice.
+		priority += 100
```

Comment:
This seems fine. But you don't actually use the generate task factor in this code, so is there a point of having it? If automatically incrementing the priority by 100 for generate tasks is enough, you can probably get rid of the factor all together and just keep this update here.

Reply:
oh you're right, this was sort of a mindless update. I do kind of  think this should be a behind-the-scenes thing that users wouldn't need to know about (although I'm not sure why they need to know about some of the other factors either) so if this seems like a reasonable way of handing generator priority then I'll remove the factor code

Reply:
@bsamek do you have an opinion? is there a reason we would still want to consider the factor? I could instead here change the priority to `priority * generateTaskFactor` if that's something we'd want to allow to be configurable

Reply:
Users do not really interact with the factors, they are only for us on the admin page. I am in favor of the factor if it is useful. Again, scheduler math, in my experience, has been super hand-wavey and it can be difficult to gauge the affect of factors.

Reply:
Maybe try some back-of-envelope calculations to compare with and without the factor.

Reply:
Well assuming we made the factor 100 then it would have a greater impact; I think it would always be below commit queue patches, generators in patches will be above probably everything else, generators in mainline builds would be below patches still but ahead of other things in the patch. I'll stick with the factor so if somehow this change is disruptive we can tweak it without having to revert.